tcbuilder/backend/tcbuild.schema.yaml

```diff
@@ -305,6 +305,9 @@ properties:
                   registry:
                     type: string
                     description: "Alternative container registry used to access container image"
+                  ca-certificate:
+                    type: string
+                    description: "CA certificate path of alternative container registry"
```

Comment:
I think should good add also a `tdxMeta` property, these are used to help integrated development environments know what show in the autocompletion. For this case I think that the IDE should show the `*.cer` files, right? So, maybe something like this:

```yaml
tdxMeta: "file;cer"
```

Reply:
I have not found any information about txdMeta format. For ```*.cer``` and ```*.crt``` files, is ```tdxMeta: "file;cer,crt"``` ok?

Reply:
Hi @franc101 yeah this is not documented yet, but you are right `file;cer,crt` should work to show the completions for all `*.cer` and `*.crt` in the workspace. Thanks.